packages/dashboard/tsconfig.json

```diff
@@ -0,0 +1,43 @@
+{
+    "compilerOptions": {
+        /* Visit https://aka.ms/tsconfig.json to read more about this file */
+        "target": "es2020",
+        "module": "es2020",
+        "jsx": "react",
+        "sourceMap": true,
+        "removeComments": true,
+
```

Comment:
Should we also add configs for `outDir` as `./dist` and `rootDir` as `./src` in the dashboard package to have better file separation by keeping all the compiled js files together?

Reply:
Sure let's add these. 👍